index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
```

Comment:
- [Optional] Kindly your logo is not part of your navigation, so make it outside the `nav` tag for sematic reasons :+1: